Fix calculation of percentage and update README

ChinYikMing opened the pull request.

jserv commented on README.md:
````diff
@@ -172,6 +172,20 @@ You can use the option with `-q` to use the output directly.
+make tool
+```
+### rv_histogram
+This is a static analysis tool for target program.
````
Improve the wording.

jserv commented on README.md:
````diff
@@ -172,6 +172,20 @@ You can use the option with `-q` to use the output directly.
+```
+It has two optional options:
+* `-a`: output the analysis in ascending order(default is descending order)
+* `-r`: output usage of registers(default is usage of instructions)
````
Give some reference output as [rv8](https://github.com/michaeljclark/rv8) does.

You may create image file(s) in `docs` directory.

jserv commented on README.md:
```diff
@@ -172,6 +172,27 @@ You can use the option with `-q` to use the output directly.
+* `-r`: output usage of registers(default is usage of instructions)
+
+_Example Instructions Histogram_
+![Instructions Hisrogram Example](/docs/images/tools/instructions_histogram.png)
```
Shorten the hyperlink. e.g., `docs/histogram-instruction.png`
Adjust the terminal background to white (or light colors) and reduce the width for proper presentation with markdown document.

ChinYikMing committed:
Fix calculation of percentage and update README

Eliminate the inaccuracy in the percentage of the output
and add some descriptions about the static analysis tool
also show some example screenshots of the tool
